tests/posix/threads_base/src/pthread.c

```diff
@@ -487,7 +484,7 @@ ZTEST(pthread, test_sched_setparam)
 	zassert_true((rc == -1 && err == ENOSYS));
 }
 
-ZTEST(pthread, test_sched_setscheduler)
+ZTEST(posix_threads_base, test_sched_setscheduler)
```

Comment:
This test belongs in the `xsi_realtime` testsuite with the `_POSIX_PRIORITY_SCHEDULING` Option.